src/cassandra/cli/load.script

```diff
@@ -8,6 +8,7 @@ CREATE COLUMN FAMILY metrics_discovery WITH column_type='Standard' AND comparato
 CREATE COLUMN FAMILY metrics_state WITH column_type='Standard' AND comparator='UTF8Type' AND key_validation_class='LongType' AND default_validation_class='LongType';
 CREATE COLUMN FAMILY metrics_metadata WITH column_type='Standard' AND comparator='UTF8Type' AND key_validation_class='UTF8Type' AND default_validation_class='BytesType';
 CREATE COLUMN FAMILY metrics_string WITH column_type='Standard' AND comparator='LongType' AND key_validation_class='UTF8Type' AND default_validation_class='UTF8Type';
+CREATE COLUMN FAMILY metrics_enum1 WITH column_type='Standard' AND comparator='LongType' AND key_validation_class='UTF8Type' AND default_validation_class='UTF8Type';
```

Comment:
Not too fond of the the column family name "metrics_enum1".  How about something like "metrics_enum_values" or "metrics_enum_hashes"?